src/components/FormInput.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
```

Comment:
Create a new `.scss` called `formInput.scss` and import it here.

Move the classes required by this component from `main.scss` to `formInput.scss`

Also move the required variables with it